src/main/java/carpet/CarpetSettings.java

```diff
@@ -299,7 +299,7 @@ public String description() {
     )
     public static String commandPerimeterInfo = "true";
 
-    @Rule(desc = "Enables /draw commands", extra = "... allows for drawing simple shapes", category = COMMAND)
+    @Rule(desc = "Enables /draw commands", extra = "... allows for drawing simple shapes or other shapes which are sorta difficult to do normally", category = COMMAND)
```

Comment:
please break up extra into two lines - otherwise it wraps weirdly in a UI

Reply:
Done